### PR TITLE
Dockerize instance validator

### PR DESCRIPTION
**Pull request by Ray725**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Dockerfile for instance validator, docs available at
```
Comment (charbull) [inline Dockerfile:1]: can you move this file to tools/

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# tools/validators/instance_validator/README.md
+
+FROM python:3
+COPY ./ ./
+ENV PYTHONPATH ./
+WORKDIR ./tools/validators/instance_validator
+RUN pip install strictyaml
+RUN pip install absl-py
+RUN pip install pyglib 
+RUN pip install pyyaml
```
Comment (charbull) [inline Dockerfile:11]: you can install setup.py instead ?

File: tools/validators/instance_validator/README.md [hunk 3]
```diff
@@ -12,3 +12,13 @@ If you want to use a modified ontology, run the following: `python3 instance_val
 When using a modified ontology, ensure you follow the folder-naming convention: `digitalbuildings/ontology/yaml`. This will allow the instance validator to rely on the new types in the ontology.
 
 Note: as of the current development stage, you must clone the entire repository and run this instance validator script from this directory.
+
+## Using Docker
+
+We provide a Docker solution for clients to run this validator on any platform.
+
+Rename your configuration file to be `config.yaml` and add it to the `tools/validators/instance_validator` folder.
```
Comment (charbull) [inline tools/validators/instance_validator/README.md:20]: are we forcing the name?

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+FROM python:3
+COPY ./ ./
+ENV PYTHONPATH ./
+WORKDIR ./tools/validators/instance_validator
+RUN pip install strictyaml
+RUN pip install absl-py
+RUN pip install pyglib 
+RUN pip install pyyaml 
+CMD ["python", "instance_validator.py", "-i", "config.yaml"]
```
Comment (charbull) [inline Dockerfile:12]: `building_config.yaml`

File: tools/validators/instance_validator/README.md [hunk 5]
```diff
@@ -12,3 +12,13 @@ If you want to use a modified ontology, run the following: `python3 instance_val
...
+
+Navigate to the `digitalbuildings` project directory then run `sudo docker build -t instance_validator_dockerized .` (with the period included).
```
Comment (charbull) [inline tools/validators/instance_validator/README.md:22]: put it in tools/

File: Dockerfile (hunk 1 above)
Comment (Ray725) [reply Dockerfile:1]: The universe generation doesn't work when the file is moved to a lower directory.

Comment (Ray725) [reply Dockerfile:1]: Done

File: Dockerfile (hunk 2 above)
Comment (Ray725) [reply Dockerfile:11]: Done

File: tools/validators/instance_validator/README.md (hunk 3 above)
Comment (Ray725) [reply tools/validators/instance_validator/README.md:20]: Yes, I can change it to be `building_config.yaml`

File: Dockerfile (hunk 4 above)
Comment (Ray725) [reply Dockerfile:12]: Changed

File: tools/validators/instance_validator/README.md (hunk 5 above)
Comment (Ray725) [reply tools/validators/instance_validator/README.md]: Done

**Comment by Ray725** [comment]
Yes, there might be a way to test the Dockerfile the way I've written it. I'll change it if that proves impossible.

**Pull request closed without merging**